front/src/utils/date.ts

```diff
@@ -52,7 +52,7 @@ export const dateTimeToIso = (inputDateTime: string) => {
  * @param msDate milliseconds date (elapsed from January 1st 1970)
  * @return an ISO 8601 date (e.g. 2024-04-25T08:30:00+02:00)
  */
-export const msToIsoDate = (msDate: number, formatDate: boolean = false) => {
+export const msToIsoDate = (msDate: number | string, formatDate: boolean = false) => {
```

Comment:
```suggestion
export const formatIsoDate = (date: number | string, formatDate: boolean = false) => {
```

Reply:
we already have a formatIsoDate in date.ts, so I rename it formatToIsoDate